### DISCUSSION TITLE
Support rotation?

### DISCUSSION THREAD

**Discussion by equal-l2** [MonitorControl/MonitorControl#1204] [Ideas]
Hi, would you consider adding a capability to rotate the orientation of a display (e.g. portrait, landscape, and flipped counterparts of them)?

Thank you for developing the fantastic software by the way!

**Comment by waydabber** [comment]
Hi there - I added this feature is available to [BetterDisplay](https://betterdisplay.pro), you can check that out.

<img width="475" alt="Screenshot 2022-11-14 at 11 54 46" src="https://user-images.githubusercontent.com/37590873/201642603-777ae9c4-1e86-49e3-9e62-2acef57422dd.png">
